src/biogeophys/PhotosynthesisMod.F90

```diff
@@ -2791,18 +2802,18 @@ subroutine PhotosynthesisHydraulicStress ( bounds, fn, filterp, &
       kcha    = 79430._r8
       koha    = 36380._r8
       cpha    = 37830._r8
-      vcmaxha = 72000._r8
-      jmaxha  = 50000._r8
-      tpuha   = 72000._r8
+      vcmaxha = 73637._r8
+      jmaxha  = 50300._r8
+      tpuha   = 73637._r8
```

Comment:
-[ ] I'm pretty sure  these should not  be changed if we're using  Kattge. @lmbirch89 can you confirm these are Leuning related changes

Reply:
Yes, those changes are for Leuning, so Kattage should have the 72k and 50k values.

Reply:
This has been done, so I'm marking this as completed. I left commented code that show the Lenuning values for reference.